security/README.md

```diff
@@ -55,3 +38,62 @@ PGP and GnuPG
 * Don't share your private key with anyone, including services like Keybase.
 * Keep at least one backup of your private key and revocation certificate in a
   secure location, such as a thumb drive.
+
+## Physical Security
+
+The following guidelines apply to how we physically secure
+our laptops and mobile devices that may contain customer or user data.
+
+* Don't leave your devices unattended in an unsecured area.
+* Install a device tracking and remote data wipe tool such as [Prey].
+
```

Comment:
Should we add:

* Lock your device when you are away from it.

On the one hand, people don't currently do this reliably, so it might be odd to document a thing that we don't do. On the other hand, we should be doing this.